.github/workflows/release-npm-packages.yml

```diff
@@ -46,9 +46,9 @@ jobs:
         id: token
         if: steps.commit.outcome == 'success'
         run: |
-          aws configure --profile token set role_arn ${{ secrets.TOKEN_ROLE }}
+          aws configure --profile token set role_arn ${{ secrets.JS_TEAM_TOKEN_ROLE }}
           aws configure --profile token set credential_source Environment
-          npm_token=$(aws secretsmanager get-secret-value --region us-west-2 --secret-id=SMITHY_PUBLISH_npmToken --query SecretString --output text --profile token)
+          npm_token=$(aws secretsmanager get-secret-value --region us-west-2 --secret-id=smithy-typescript-npm-token --query SecretString --output text --profile token)
```

Comment:
this step was tested in https://github.com/smithy-lang/smithy-typescript/pull/1493